### ISSUE TITLE
Fix: letterId undefined

### ISSUE THREAD

**Issue by raygdev** [raygdev/simply-lit#11]
in get route that takes the letter id as a param, the letter id is undefined not allowing for result from schema when searching.

**Comment by raygdev** [comment]
Also, no response is being sent for the result of the letters document being found from that controller.

**Comment by CampbellSDC** [comment]
Is this issue resolved? I assigned myself to it and then saw that it shows there is a pull request that would close this issue. Just wanted to make sure

**Comment by raygdev** [comment]
@CampbellSDC yeah lol. I made a pull request and merged it to another pull request for the other issue that is there. It's just waiting on your review and approval.

**Comment by raygdev** [comment]
And my apologies... I did that a bit out of order and probably should have self assigned them.
